src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -0,0 +1,34 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
+
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.command.Subsystem;
+
+/**
+ * Add your docs here.
+ */
+public class DriveTrain extends Subsystem {
+  public static void FlyByWire (TalonSRX starboard, TalonSRX port, Joystick driverstick)
+  {
+    // Gets the position of the joystick and moves the motors accordingly.
+    Double thro = driverstick.getRawAxis(1);
```

Comment:
Try adding in another line to get the steerage or the yaw, you can use the code in joe-drivetrain as an example.
The axis for yaw is 2